### DISCUSSION TITLE
This application failed to start because it could not find or load the Qt platform plugin "wayland;xcb" in "".

### DISCUSSION THREAD

**Discussion by projihadi** [JaKooLit/Fedora-Hyprland#136] [Q&A]
❯ vktablet
This application failed to start because it could not find or load the Qt platform plugin "wayland;xcb"
in "".

Available platform plugins are: xcb.

Reinstalling the application may fix this problem.
[1]    7496 IOT instruction (core dumped)  vktablet

How do I fix this?

**Comment by JaKooLit** [comment]
https://github.com/jlam55555/veikk-linux-driver

maybe this? not sure if its for digitizers?

**Reply by projihadi** [reply]
My drawing tablet is not being detected in hyprland. I tried running the official veik tablet drivers from https://veikk.com/route/v2/support/download but it does not launch on hyprland after installing but works perfectly on gnome. I created an issue on the official hyprland repository but they told me that:

You might need to add an exec-once in your Hyprland config for the driver daemon because the rpm package lists a vktablet.desktop in /etc/xdg/autostart/ which would be started alongside generic desktop environments (that Hyprland is not). The device should then be listed when viewing hyprctl devices in your terminal of choice.

So I set vktablet(the driver daemon for the tablet) as exec once in env variables but it would not run after startup and when I try running vktablet manually through the terminal it gives me the error mentioned above.
